common/views/components/Modal/Modal.test.tsx

```diff
@@ -38,20 +38,20 @@ describe('Modal', () => {
     expect(document.activeElement).not.toEqual(openButton);
   });
 
-  it('should focus the close button when opened', () => {
+  it('should focus the close button when opened', async () => {
     renderComponent();
     const openButton = screen.getByText(/^Open modal window$/i);
-    userEvent.click(openButton);
+    await userEvent.click(openButton);
     const closeButton = screen.getByTestId('close-modal-button');
-    expect(document.activeElement).toEqual(closeButton);
+    await waitFor(() => expect(document.activeElement).toEqual(closeButton));
```

Comment:
God this brings back (bad) memories - you might prefer to use `findBy` instead of `waitFor` as recommended in this very useful article by the author of testing-library https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-waitfor-to-wait-for-elements-that-can-be-queried-with-find

Reply:
changes made